Chiazokam/Politico
# #163498721 create the user login endpoint

#### What does this PR do?
This PR tests and creates the user login Endpont

#### Description of Task to be completed?
- write tests for every test case
- write validators for the user input
- create the user login endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-user-login-163498721`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `POST` request to `http://localhost:3000/api/v1/auth/login `
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint payload should be structured like this- 
`
{
            "email": "daun@mabel.com",
            "password": "lilian"
        }
`
#### What are the relevant pivotal tracker stories?
[#163498721](https://www.pivotaltracker.com/story/show/163498721)

### Images
![image](https://user-images.githubusercontent.com/26940294/51833925-12419600-22f9-11e9-8aab-272502a1c6f1.png)
